packages/components/src/components/message/message.tsx

```diff
@@ -7,8 +7,8 @@ import { cn } from '~/lib/utils';
 const messageVariants = cva('flex w-full gap-x-2.5 justify-start p-3 text-base', {
   variants: {
     variant: {
-      success: 'bg-green-300/20 [&>svg]:text-green-300',
-      error: 'bg-red-100/20 [&>svg]:text-red-100',
+      success: 'bg-success/[.15] [&>svg]:text-success',
+      error: 'bg-error/[.15] [&>svg]:text-error-secondary',
```

Comment:
this will render darker green color on success variant

Reply:
Yes, that is fine, we'll update the Figma designs.

Reply:
Actually, not sure if you saw my comment but this is what we want from now on: https://github.com/bigcommerce/catalyst/pull/622#discussion_r1515244682

Reply:
changed based on recomended code on `Change Message Component Background Colors Based on Design Comment` commit

Reply:
Perfect, thanks!